store/store.go

```diff
@@ -1162,12 +1165,13 @@ func currentSnap(cs *RefreshCandidate) *currentSnapJSON {
 		SnapID:   cs.SnapID,
 		Channel:  channel,
 		Epoch:    cs.Epoch,
-		Revision: revision,
+		Revision: cs.Revision.N,
 		// confinement purposely left empty
 	}
 }
 
-func (s *Store) checkCandidates(currentSnaps []*currentSnapJSON, user *auth.UserState) ([]snapDetails, error) {
+// query the store for the latest revisions of currentSnaps
```

Comment:
latest revision conveys the wrong idea because as we know there's no orderning,  offered revisions?